scripts/run_backend_tests.py

```diff
@@ -284,9 +368,16 @@ def main(args=None):
             time.sleep(3)
             python_utils.PRINT('Redirecting to its corresponding test file...')
             all_test_targets = [parsed_args.test_target + '_test']
+    elif parsed_args.test_shard:
+        validation_error = _check_shards_match_tests(
```

Comment:
This will validate the entire test files in the codebase for all the shards? How expensive is it (in secs.)?

Reply:
This takes around 30 secs on my (slow) computer. The `_check_shards_match_tests` does the more expensive operations to figure out which shard is duplicated or which test is missing from the shards only when we already know there's a problem. This keeps it pretty efficient in the happy cases